dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/Event.java

```diff
@@ -47,6 +47,9 @@
 public class Event
     extends BaseLinkableObject
 {
+
+    private String uid;
```

Comment:
This is for exporting, in class JdbcEventStore line 134. it is less complicated if I use the uid instead of having to check the event attribute which can be uid or code or null.